src/components/ArtistPieChart/ArtistPieChart.tsx

```diff
@@ -0,0 +1,292 @@
+  const [brand2, setBrand2] = useState('');
+  const [brand3, setBrand3] = useState('');
+  const [brand4, setBrand4] = useState('');
+  const [name1, setName1] = useState('');
+  const [name2, setName2] = useState('');
+  const [name3, setName3] = useState('');
+  const [name4, setName4] = useState('');
+  const [match1, setMatch1] = useState('');
+  const [match2, setMatch2] = useState('');
+  const [match3, setMatch3] = useState('');
+  const [match4, setMatch4] = useState('');
+  const [img1, setImg1] = useState('');
+  const [img2, setImg2] = useState('');
+  const [img3, setImg3] = useState('');
+  const [img4, setImg4] = useState('');
+  const [age1, setAge1] = useState('');
+  const [age2, setAge2] = useState('');
+  const [age3, setAge3] = useState('');
+  const [age4, setAge4] = useState('');
+  const [gender1, setGender1] = useState('');
+  const [gender2, setGender2] = useState('');
+  const [gender3, setGender3] = useState('');
+  const [gender4, setGender4] = useState('');
+  const [location1, setLocation1] = useState('');
+  const [location2, setLocation2] = useState('');
+  const [location3, setLocation3] = useState('');
+  const [location4, setLocation4] = useState('');
+
+  const getData = async () => {
+    const data = await getRecommendedArtists('871717');
+    const { artists } = data;
+    const [artist1, artist2, artist3, artist4] = artists;
+    // eslint-disable-next-line
+    console.log(artist1);
+    setName1(artist1.artistName);
+    setName2(artist2.artistName);
+    setName3(artist3.artistName);
+    setName4(artist4.artistName);
+    setBrand1(getBrands(artist1.matchAttributes.associatedBrands));
+    setBrand2(getBrands(artist2.matchAttributes.associatedBrands));
+    setBrand3(getBrands(artist3.matchAttributes.associatedBrands));
+    setBrand4(getBrands(artist4.matchAttributes.associatedBrands));
+    setMatch1(artist1.matchPercentage);
+    setMatch2(artist2.matchPercentage);
+    setMatch3(artist3.matchPercentage);
+    setMatch4(artist4.matchPercentage);
+    setImg1(artist1.artistImage);
+    setImg2(artist2.artistImage);
+    setImg3(artist3.artistImage);
+    setImg4(artist4.artistImage);
+    setAge1(getAge(artist1.matchAttributes.age));
+    setAge2(getAge(artist2.matchAttributes.age));
+    setAge3(getAge(artist3.matchAttributes.age));
+    setAge4(getAge(artist4.matchAttributes.age));
+    setGender1(getGender(artist1.matchAttributes.gender));
+    setGender2(getGender(artist2.matchAttributes.gender));
+    setGender3(getGender(artist3.matchAttributes.gender));
+    setGender4(getGender(artist4.matchAttributes.gender));
+    setLocation1(getLocations(artist1.matchAttributes.venues));
+    setLocation2(getLocations(artist2.matchAttributes.venues));
+    setLocation3(getLocations(artist3.matchAttributes.venues));
+    setLocation4(getLocations(artist4.matchAttributes.venues));
+  };
+  useEffect(() => {
+    getData();
+  }, []);
+  return (
+    <div className="container">
```

Comment:
think of having a loop instead with classes being injected dymanically rather than duplicating the html content throughtout